pkg/helm/values/k3s.go

```diff
@@ -100,11 +107,20 @@ func addCommonReleaseValues(values string, chartOptions *helm.ChartOptions) (str
 serviceCIDR: ##CIDR##
 storage:
   size: 5Gi`
-	if chartOptions.DisableIngressSync {
+	if chartOptions.DisableIngressSync && chartOptions.Isolated {
+		values += `
+syncer:
+  extraArgs: ["--disable-sync-resources=ingresses", "--enforce-pod-security-standard=baseline"]`
+	} else if chartOptions.DisableIngressSync {
 		values += `
 syncer:
   extraArgs: ["--disable-sync-resources=ingresses"]`
+	} else if chartOptions.Isolated {
+		values += `
+syncer:
+  extraArgs: ["--enforce-pod-security-standard=baseline"]`
```

Comment:
instead of doing it like this, rather do it in the helm chart as soon as `isolation.enabled` is true. Also please add an extra option `isolation.podSecurityStandard: baseline` to define which standard to use